.github/workflows/ci.yml

```diff
@@ -39,16 +39,3 @@ jobs:
       - name: Test
         id: npm-ci-test
         run: npm run ci-test
-
```

Comment:
assuming this is re-inserted after this PR is merged ([post creating an action as per docs](https://github.com/actions/typescript-action#validate-the-action))

Reply:
@seungpark I actually removed it because the CI will always fail here because this action depends on another step. I guess I could refactor it so that it would be easier to use the output from a previous step, but it reads a file and I am not sure of a way to do that using inputs/outputs. 

I will make another story to get integration testing working again. For the time being, I am disabling this step.